ts/components/buttons/CopyToClipboardButton.tsx

```diff
@@ -56,13 +54,12 @@ type CopyToClipboardIconProps = Omit<SessionIconButtonProps, 'children' | 'onCli
   CopyProps;
 
 export const CopyToClipboardIcon = (props: CopyToClipboardIconProps & { copyContent: string }) => {
-  const { copyContent, onCopyComplete, hotkey = false, showToast = true } = props;
+  const { copyContent, onCopyComplete, hotkey = false } = props;
 
   const onClick = () => {
     clipboard.writeText(copyContent);
-    if (showToast) {
-      ToastUtils.pushCopiedToClipBoard();
-    }
+    ToastUtils.pushCopiedToClipBoard();
```

Comment:
Just confirming you wanted to revert this back to the original behaviour?

Reply:
indeed